examples/gno.land/p/sys/vals/poa/poa.gno

```diff
@@ -0,0 +1,106 @@
+package poa
```

Comment:
The `p/sys/vals/poa` should be located under `p/nt/poa`, while `p/sys/vals/types` should be located under `p/sys/vals`.

`p/sys` should remain minimal.

`p/nt` is a suitable location for the NewTendermint team to release reusable libraries, such as `poa (v1)`.

Reply:
Applied in:

5371852